.gitignore

```diff
@@ -17,4 +17,5 @@ dist
 .cache
 .npmrc
 examples/**/build/
-.scannerwork/
\ No newline at end of file
+.scannerwork/
+.vscode/
```

Comment:
I think it's a good practice to commit your vscode config. In that case, anyone that clone your project will have the same configuration.

Reply:
I don't know. In this folder there are just my personnals fews config. I don't think that every one want to get my own settings.

Reply:
No, in your .vscode folder in your project, you have to specify the configuration for this project. You own settings are store in user settings.

Reply:
I agree with @johnmeunier ^^